receiver/datadogreceiver/translator.go

```diff
@@ -88,7 +89,8 @@ func toTraces(payload *pb.TracerPayload, req *http.Request) ptrace.Traces {
 			if span.Error > 0 {
 				newSpan.Status().SetCode(ptrace.StatusCodeError)
 			}
-
+			newSpan.Attributes().PutStr("datadog.span.id", strconv.FormatUint(span.SpanID, 10))
```

Comment:
I would be nice to have these strings in an enum like semconv.  DD changes these all the time and I suspect that there will be many changes like this in the future.  Why not make a data structure to hold these strings and then loop over it dynamically? That way in the future we can just add a new entry to the enum.